spire/k8s/crd/spire.spiffe.io_clusterfederatedtrustdomains.yaml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
These manifests are mostly copy/paste with minor changes.
I will update the README accordingly.